src/DvfHelpers.php

```diff
@@ -50,8 +48,7 @@ public function transformMachineName($string) {
    *   The link.
    */
   public function getHelpPageLink($template_name) {
-    $link = Link::fromTextAndUrl('Help', Url::fromUserInput($this->helpPageBasePath . $template_name));
-    return new FormattableMarkup('<span class="dvf-admin-popup">' . $link->toString() . ' &#x29c9;</span>', []);
+    return new FormattableMarkup('<span class="dvf-admin-popup"><a href="/dvf/help/' . $template_name . '">Help</a> &#x29c9;</span>', []);
```

Comment:
Also just a suggestion if you want to use render array instead of markup. https://www.drupal.org/docs/drupal-apis/render-api/render-arrays#:~:text=This%20provides%20enormous%20flexibility%20in,Drupal's%20equivalent%20of%20the%20DOM.